service/controller/resource/ensurecpcrs/entityid/generator.go

```diff
@@ -0,0 +1,48 @@
+package entityid
```

Comment:
Does this need to be in a separate package?

Reply:
It does not but I just copied it as is from `api` where it is separate.